openshift/release/generate-release.sh

```diff
@@ -15,15 +14,15 @@ else
 fi
 
 # the core parts
-resolve_resources config/ $output_file $image_prefix $release
+resolve_resources config/ $output_file $image_prefix $tag
 
 # in-memory-channel (ccp)
 # TODO: remove in 0.8.0 - because the deprecated CCP is removed by than
```

Comment:
It's 0.8.0 - act on the comment or remove it?

Reply:
that was always wrong, and did cause wrong `:ci` tags